Tests/conf.json

```diff
@@ -1123,6 +1123,11 @@
             "fromversion": "5.0.0",
             "has_api": false
         },
+        {
+            "integrations": "Netmiko",
+            "playbookID": "Netmiko_test",
+            "instance_names": "Netmiko"
```

Comment:
Do we have credentials for this integration?
If yes, let's remove the `instance_names` field as we only have one set of credentials,
And @dantavori please update them in Google Secret.
If not, we can remove the TPB definition from the conf.json file, and from the yml as well.

Reply:
we should have an instance, I'm checking this today 👍🏽